.eslintignore

```diff
@@ -23,4 +23,4 @@ lighthouse-core/report/templates/report-templates.js
 lighthouse-core/report/partials/templates/report-partials.js
 
 # Generated files
-plots/out/**
+plots/out*
```

Comment:
Personally, i've been taking old `out` folders and renaming them to `out-old-thevergeruns` etc. cc @wwwillchen in case he has a preference for handling measure runs differently.